wandb/sdk/wandb_init.py

```diff
@@ -449,12 +449,67 @@ def init(
     id=None,
     settings: Union[Settings, Dict[str, Any], None] = None,
 ) -> RunBase:
-    """Initialize a wandb Run.
-
+    """Initialize W&B
+    If called from within Jupyter, initializes a new run and waits for a call to
+    `wandb.log` to begin pushing metrics.  Otherwise, spawns a new process
+    to communicate with W&B.
```

Comment:
We don't need to the callout of waiting for wandb.log.  The only difference in jupyter now is that we're only streaming metrics when cells are running, probably don't need to call out the difference.